### DISCUSSION TITLE
Use Pagination without dots is possible?

### DISCUSSION THREAD

**Discussion by realyuki** [mantinedev/mantine#6016] [Q&A]
I would like to use pagination like this:

![image](https://github.com/mantinedev/mantine/assets/25220728/a5c18e3a-ece1-4d6c-9473-a10e11e040d2)

Is there a way to use like this?

**Comment by rtivital** [comment]
https://codesandbox.io/s/mantine-react-template-forked-ldmdss?file=/src/App.tsx

**Reply by realyuki** [reply]
Wow, I'll ask you one more question. 
These are all spread out, but I would like to expose 10 at a time without dots. Is it possible?
